### PR TITLE
Changed HDM Version to 2.2 in release notes and copyright notice.

### PR DESCRIPTION
**Pull request by Mona2012**

### CODE REVIEW

File: pages/04.HDM Doc 2.2/20.copyright/default.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: 'Copyright Notice'
 ---
 
-Version: 2.2.1
+Version: 2.2
 Publish Date: August, 2020
```
Comment (kgcb) [inline pages/04.HDM Doc 2.2/20.copyright/default.md]: August, 2020 -> June, 2021

Comment (Mona2012) [reply pages/04.HDM Doc 2.2/20.copyright/default.md]: Resolved..

**Review by kgcb — APPROVED** [review]
Looks good to me.

**Review by ayyajpio — APPROVED** [review]
Looks good to me.

**Pull request merged**